docs/reference/delegated-properties.md

```diff
@@ -85,41 +85,41 @@ For a **mutable** property (a *var*{:.keyword}), a delegate has to _additionally
  
 ## Standard Delegates
 
-The `kotlin.properties.Delegates` object from the standard library provides factory methods for several useful kinds of delegates.
+The Kotlin standard library provides factory methods for several useful kinds of delegates.
 
 ### Lazy
 
-`Delegates.lazy()` is a function that takes a lambda and returns a delegate that implements a lazy property: 
+`lazy()` is a function that takes a lambda and returns a delegate that implements a lazy property:
 the first call to `get()` executes the lambda passed to `lazy()` and remembers the result, 
 subsequent calls to `get()` simply return the remembered result. 
 
 
 ``` kotlin
-import kotlin.properties.Delegates
- 
-val lazy: String by Delegates.lazy {
+val lazyValue: String by lazy {
     println("computed!")
     "Hello"
 }
 
 fun main(args: Array<String>) {
-    println(lazy)
-    println(lazy)
+    println(lazyValue)
+    println(lazyValue)
 }
 ```
 
 
-If you want **thread safety**, use `blockingLazy()`: it works the same way, but guarantees that the values will be 
-computed only in one thread, and that all threads will see the same value.
+If you want **thread safety**, use the overload of `lazy()` that takes a lock as a parameter or pass `LazyThreadSafetyMode.SYNCHRONIZED`.
```

Comment:
SYNCHRONIZED is a default value. So you have to opt-out from thread safety, rather than opt-in for it.